main/test/ca/uwaterloo/flix/language/phase/TestWeeder.scala

```diff
@@ -357,6 +357,86 @@ class TestWeeder extends AnyFunSuite with TestUtils {
     expectError[WeederError.IllegalEscapeSequence](result)
   }
 
+  ignore("IllegalExtPattern.01") {
+    val input =
+      """
+        |def f(): Int32 = ematch xvar A(1) {
+        |    case 1 => 1
+        |}
+        |""".stripMargin
+    val result = compile(input, Options.TestWithLibNix)
+    expectError[WeederError.IllegalExtPattern](result)
+  }
+
+  ignore("IllegalExtPattern.02") {
+    val input =
+      """
+        |def f(): Int32 = ematch xvar A(1) {
+        |    case A.B(2) => 1
+        |}
+        |""".stripMargin
+    val result = compile(input, Options.TestWithLibNix)
+    expectError[WeederError.IllegalExtPattern](result)
+  }
+
```

Comment:
These have been ignore for now, but will be fixed by https://github.com/flix/flix/issues/11162#issuecomment-3174170693